monkey/monkey_island/cc/server_config.json

```diff
@@ -1,4 +1,4 @@
 {
-  "server_config": "standard",
+  "server_config": "testing",
```

Comment:
Do we want that?

Reply:
Well, we needed it for Travis UTs to work, but I agree it's preferable to keep it `standard` as the default. 

Now the default is `standard` and I've added a script + build stage in Travis for changing it to `testing`:
```
0.05s$ python monkey/monkey_island/cc/set_server_config.py testing
Config file path: monkey/monkey_island/cc/server_config.json
Writing the following config: {
    "server_config": "testing",
    "deployment": "develop"
}
```
(From https://travis-ci.com/guardicore/monkey/jobs/250116323)